README.md

```diff
@@ -79,7 +79,6 @@ If you want to implement your own source, it must match [RegistrySource interfac
 | VALIDATOR_REGISTRY_LIDO_SOURCE_SQLITE_CACHE_PATH | Validators registry lido source sqlite cache path. It makes sense to change default value if you set `VALIDATOR_REGISTRY_SOURCE` to `lido` | false        | ./docker/validators/lido_mainnet.db     |
 | SYNC_PARTICIPATION_DISTANCE_DOWN_FROM_CHAIN_AVG  | Distance (down) from Blockchain Sync Participation average after which we think that our sync participation is bad                         | false        | 0                                       |
 | SYNC_PARTICIPATION_EPOCHS_LESS_THAN_CHAIN_AVG    | Number epochs after which we think that our sync participation is bad and alert about that                                                 | false        | 3                                       |
-| ATTESTATION_MAX_INCLUSION_IN_BLOCK_DELAY         | Maximum inclusion delay after which we think that attestation is bad                                                                       | false        | 5                                       |
```

Comment:
Part of old algorithm. Now consider that max inclusion delay, after which we say that attestation is missed = 32. Because after that number attestation has no value

Reply:
Ok, you've changed the default value, why to remove the ability to configure the value?

Reply:
I don't know a case where we can change it value and not shoot in the foot. This is inconsistent with spec and is likely to confuse the user

Reply:
Ah, no question if it follows a spec. Just leave the trace, please!

Reply:
done